sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample1_RecognizeReceipts.md

```diff
@@ -0,0 +1,74 @@
+# Recognize receipts
+
+This sample demonstrates how to recognize and extract common fields from US receipts, using a pre-trained receipt model.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `FormRecognizerClient`
+
+To create a new `FormRecognizerClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Recognize receipts from a URI
```

Comment:
I would suggest flipping the ordering and have recognize receipts from a file first since I believe it's more used

Reply:
Issue https://github.com/Azure/azure-sdk-for-net/issues/11508